core/app/api/v0/c/[communitySlug]/site/[...ts-rest]/route.ts

```diff
@@ -319,6 +319,27 @@ const manuallyParsePubFilterQueryParams = (url: string, query?: Record<string, a
 const handler = createNextHandler(
 	siteApi,
 	{
+		forms: {
+			get: async ({ params }) => {
+				const { user, community } = await checkAuthorization({
+					token: { scope: ApiAccessScope.community, type: ApiAccessType.read },
+					cookies: true,
+				});
+
+				const form = await getForm({
+					id: params.formId,
+					communityId: community.id,
+				}).executeTakeFirst();
+
+				if (!form) {
+					return { status: 404, body: "Form not found" };
+				}
+				return {
+					status: 200,
+					body: form,
+				};
+			},
+		},
```

Comment:
This isn't actually used, I just realized it's here. It was going to be for the form switcher but it turns out that's also not going to be necessary so I'll remove it